packages/harmony/src/components/button/FollowButton/FollowButton.tsx

```diff
@@ -35,7 +38,9 @@ export const FollowButton = (props: FollowButtonProps) => {
     onUnfollow,
     onFollow,
     size = 'default',
-    as: ignoredAs,
+    onChange,
+    type = 'button',
+    checked,
     ...inputProps
```

Comment:
so to do the typing properly i think you'd do

{...inputProps} = props
const {type} = inputProps
...
Now that type is connected to inputProps, when you check type, itll change inputProps type so you dont have to cast